smg-automotive/cookie-consent-pkg
# feat: init implementation

References [DM-510](https://autoricardo.atlassian.net/browse/DM-510)

## Motivation and context

To share some logic between all the projects we have to include OneTrust, this package was created.

## Before

No package

## After

Package

## How to test

New stack: https://github.com/smg-automotive/next-example/pull/105 (check the print on the home page)
Old stack: https://github.com/as24ch/www/pull/2087 ( check the print on the footer)

Make sure to clean your cookies first :)
